test/saithrift/tests/sail2.py

```diff
@@ -380,6 +380,175 @@ def runTest(self):
+
+                exp_pkt = simple_tcp_packet(eth_dst='00:11:11:11:11:11',
+                                            eth_src=src_mac,
+                                            ip_dst=ip_dst,
+                                            ip_src=ip_src,
+                                            tcp_sport=sport,
+                                            tcp_dport=dport,
+                                            ip_id=109,
+                                            ip_ttl=64)
+
+                send_packet(self, 4, str(pkt))
+                rcv_idx = verify_any_packet_any_port(self, [exp_pkt], [0, 1, 2, 3])
+                count1[rcv_idx] += 1
+                laglist1.append(rcv_idx)
+                sport += 1
+                dport += 1
+        
+            print ("The distribution of packets with default hash seed value:",count1)
+
+            attr_value = sai_thrift_attribute_value_t(u32=lag_hashseed_value)
+            attr = sai_thrift_attribute_t(id= SAI_SWITCH_ATTR_LAG_DEFAULT_HASH_SEED, value=attr_value)
+            self.client.sai_thrift_set_switch_attribute(attr)
+                
+            #sending packets again		
+            count2 = [0, 0, 0, 0]
+            laglist2=list()
+            max_itrs = 101
+            src_mac_start = '00:22:22:22:22:'
+            ip_src_start = '192.168.12.'
+            ip_dst_start = '10.10.10.'
+            dport = 0x80
+            sport = 0x1234
+			
+            print ("sending 100 packets to verify the order/sequence of distribution ")
+                        
+            for i in range(0, max_itrs):
+	        src_mac = src_mac_start + str(i % 99).zfill(2)
+                ip_src = ip_src_start + str(i % 99).zfill(3)
+                ip_dst = ip_dst_start + str(i % 99).zfill(3)    
+                                
+                pkt = simple_tcp_packet(eth_dst='00:11:11:11:11:11',
+                                        eth_src=src_mac,
+                                        ip_dst=ip_dst,
+                                        ip_src=ip_src,
+                                        tcp_sport=sport,
+                                        tcp_dport=dport,
+                                        ip_id=109,
+                                        ip_ttl=64)
+
+                exp_pkt = simple_tcp_packet(eth_dst='00:11:11:11:11:11',
+                                            eth_src=src_mac,
+                                            ip_dst=ip_dst,
+                                            ip_src=ip_src,
+                                            tcp_sport=sport,
+                                            tcp_dport=dport,
+                                            ip_id=109,
+                                            ip_ttl=64)
+
+                send_packet(self, 4, str(pkt))
+                rcv_idx = verify_any_packet_any_port(self, [exp_pkt], [0, 1, 2, 3])
+                count2[rcv_idx] += 1
+                laglist1.append(rcv_idx)
```

Comment:
The rcv_idx need to be saved in different list (laglist2) for comparing the order of distribution before after changing the hash value.